hack/local-up-cluster.sh

```diff
@@ -110,6 +110,8 @@ RKT_STAGE1_IMAGE=${RKT_STAGE1_IMAGE:-""}
 CHAOS_CHANCE=${CHAOS_CHANCE:-0.0}
 CPU_CFS_QUOTA=${CPU_CFS_QUOTA:-false}
 ENABLE_HOSTPATH_PROVISIONER=${ENABLE_HOSTPATH_PROVISIONER:-"false"}
+# current pvclaimbinder-sync-period default is 10m, which causes pvc->pv bindings to be noticeably slow
+CLAIM_BINDER_SYNC_PERIOD=${CLAIM_BINDER_SYNC_PERIOD:-"30s"} # 30s default results in ~25x reduction in binding time where the claim is created before the matching pv
```

Comment:
Having `CLAIM_BINDER_SYNC_PERIOD` as override is a good idea. However, having a *different* default (30s) in Kubernetes launched by `local-up-cluster.sh` is IMO confusing. Kubernetes should have the same default (10 minutes) regardless the way how it's started. Otherwise developers (like me) will see everything is bound quickly, while people in production would have different experience.